opsdroid/connector/matrix/connector.py

```diff
@@ -234,8 +234,6 @@ async def connect(self):
                 "Configuration for the matrix connector should specify mxid and password or access_token."
             )  # pragma: no cover
 
-        self.connection.sync_token = None
```

Comment:
I am not sure why these changes from #1752 are in this diff :confused:

Reply:
I will try to remove this from this PR. I was merging the master branch to this branch, but my fat fingers actually rebased the branch instead of merging.

Reply:
This should be removed now on the last commit